packages/fake/lib/plugin.js

```diff
@@ -5,12 +5,23 @@ import B from 'bluebird';
 
 
 export default class FakePlugin extends BasePlugin {
+  constructor (pluginName, pluginArgs = {}) {
+    super(pluginName, pluginArgs);
+  }
+
+  async getFakeCliArgs () {
+    await B.delay(1);
```

Comment:
what is the reason to have the delay?

Reply:
to emulate it actually doing something and ensure there is genuine async behavior going on. it's not necessary but i've done this in the past with these fake things too.